libs/blocks/modal/modal.js

```diff
@@ -162,3 +164,15 @@ window.addEventListener('hashchange', (e) => {
     if (details) getModal(details);
   }
 });
+
+function sendViewportDimensionsToiFrame(source) {
+  const viewportWidth = Math.max(document.documentElement.clientWidth || 0, window.innerWidth || 0);
+  source.postMessage({ mobileMax: MOBILE_MAX, tabletMax: TABLET_MAX, viewportWidth }, '*');
+}
+
+window.addEventListener('message', (t) => {
```

Comment:
This is listening for the `message` event which would only fire if the page is inside an iFrame.  Is that the intent?